kedro_azureml/datasets.py

```diff
@@ -57,3 +66,22 @@ def _describe(self) -> Dict[str, Any]:
             "dataset_name": self.dataset_name,
             "path": self._get_target_path(),
         }
+
+
+class KedroAzureRunnerDistributedDataset(KedroAzureRunnerDataset):
+    def _load(self):
```

Comment:
Consider using here libraries that support backoff like https://github.com/litl/backoff or https://github.com/jd/tenacity. They do not print any extra deps and allow you to use decorator here

Reply:
Thanks, `backoff` seems nice.